### DISCUSSION TITLE
为什么会取taro这个名字，外星人头像有什么特别意义吗？

### DISCUSSION THREAD

**Discussion by BoBoDinachen** [NervJS/taro#12690] [Q & A]
刚开始学习taro，这是我的疑惑

**Comment by ModyQyW** [comment]
<img width="869" alt="image" src="https://user-images.githubusercontent.com/31850793/196878079-0fba8539-4565-4d42-a3e0-bcef130346ed.png">

**Comment by luckyadam** [comment]
Taro 是「凹凸实验室」发起的项目，「凹凸 Man」谐音奥特曼，所以项目采用奥特曼的名字作为代号
